### PR TITLE
Tg   binary upgrade, interpret data, fees rewards parameters

### PR DESCRIPTION
**Pull request by TommyGarch**

### CODE REVIEW

**Review by jiajames — APPROVED** [review]

File: pages/getting_started/getting_started.json
Comment (jaredvu) [inline pages/getting_started/getting_started.json:1]: change the name of this file to `_meta.json`

File: pages/guides/orderbook_stream.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ The initial implementation only contains orders but not trades. Also note that b
 
 This feature can be enabled via a command line flag (--grpc-streaming-enabled=true) when starting your full node. This feature can only be used on non validating full nodes and when grpc is also enabled.
 
+Further, please ensure your full node binary is v3.0.1, or v4.0.1 when it becomes mandatory on April 8th (subject to governance vote).
```
Comment (jaredvu) [inline pages/guides/orderbook_stream.md:13]: nits:
* put version in backticks?
* Add year to date

File: pages/trading/_meta.json [hunk 2]
```diff
@@ -1,5 +1,4 @@
 {
-  "Rewards_and_Parameters": "Rewards, Fees and Parameters",
```
Comment (jaredvu) [inline pages/trading/_meta.json:2]: Is the associated .md deleted?

**Review by jaredvu — CHANGES_REQUESTED** [review]
small items to fix

**Review by jiajames — APPROVED** [review]

**Review by jaredvu — APPROVED** [review]

**Pull request merged**